### PR TITLE
update intellisense file

### PR DESCRIPTION
**Pull request by ranjeshj**
Update intellisense file.

### CODE REVIEW

File: build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml [hunk 1]
```diff
@@ -4,110 +4,206 @@
     <name>Microsoft.UI.Xaml</name>
   </assembly>
   <members>
-    <member name="T:Microsoft.UI.Xaml.Automation.Peers.AnimatedVisualPlayerAutomationPeer" />
+    <member name="T:Microsoft.UI.Xaml.Automation.Peers.AnimatedVisualPlayerAutomationPeer">
+      <summary>Exposes AnimatedVisualPlayer" types to Microsoft UI Automation.</summary>
```
Comment (Felix-Dev) [inline build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml:8]: Why the single " (can be seen throughout the file)? (Looks to me like it was forgotten to be removed.)

Comment (StephenLPeters) [reply build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml:8]: I have the same question @ranjeshj

Comment (ranjeshj) [reply build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml:8]: Looks like an issue with the intellisense pipeline. following up.

Comment (ranjeshj) [reply build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml]: This is fixed now.

**Review by StephenLPeters — APPROVED** [review]

**Pull request merged**